var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -86,6 +86,7 @@ class Gcc(AutotoolsPackage):
     depends_on('isl@0.15:', when='@6:')
     depends_on('zlib', when='@6:')
     depends_on('gnat', when='languages=ada')
+    depends_on('gnat', when='languages=all')
```

Comment:
personally I am against propagating this `all` into packages. I think it should be removed altogether from `gcc`.